build-logic/publishing/build.gradle.kts

```
@@ -10,5 +10,6 @@ dependencies {
     implementation(project(":basics"))
     implementation(project(":jvm"))
     implementation("dev.sigstore.build-logic:gradle-plugin")
+    implementation("dev.sigstore:sigstore-gradle-sign-plugin:0.5.0")
```

```suggestion
    implementation("dev.sigstore.sign:dev.sigstore.sign.gradle.plugin:0.5.0")
```

This way the artifact name and group matches the plugin id